testkit/src/main/java/com/example/testkit/ReadSideTestDriver.java

```diff
@@ -67,14 +68,15 @@ public ReadSideTestDriver(Injector injector, Materializer materializer) {
   public <Event extends AggregateEvent<Event>> CompletionStage<Done> feed(Event e, Offset offset) {
     AggregateEventTagger<Event> tag = e.aggregateTag();
 
-    List<Pair<ReadSideProcessor.ReadSideHandler<?>, Offset>> list = processors.get(tag.eventType());
+    List<Pair<ReadSideHandler<?>, Offset>> list = processors.get(tag.eventType());
 
     if (list == null) {
       throw new RuntimeException("No processor registered for Event " + tag.eventType().getCanonicalName());
     }
 
-    List<CompletionStage<Done>> stages = list.stream().map(pHandlerOffset -> {
-          Flow<Pair<Event, Offset>, Done, ?> flow = ((ReadSideProcessor.ReadSideHandler<Event>) pHandlerOffset.first()).handle();
+    List<CompletionStage<?>> stages = list.stream().map(pHandlerOffset -> {
+      @SuppressWarnings("unchecked") ReadSideHandler<Event> handler = (ReadSideHandler<Event>) pHandlerOffset.first();
```

Comment:
The only change here was to add an explicit `@SuppressWarnings("unchecked")` annotation, so we don't get a warning here. Unfortunately, there is no way to write this code in Java that does not produce a warning, but the class itself ensures the safety of this cast. Using the annotation here means we don't see the warning, but only for this specific variable.

All of the other changes in this file are just refactoring.